.github/workflows/build.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
   unit-test:
     strategy:
       matrix:
-        go: ['1.19', '1.20', '1.21']
+        go: ['1.19', '1.20', '1.21', '1.22']
```

Comment:
Can we drop 1.19 and 1.20 given they are already EOL?

Reply:
May need to update go.mod min supported version to 1.21

Reply:
Pushed. PTAL.